specs/cli/plugin-add.md

```diff
@@ -38,6 +45,60 @@ For `apimanifest`, the generated file will be named `{plugin-name}-apimanifest.j
 | publisherEmail | Defaults to the contact email from the OpenAPI document. If the contact email is not available, it defaults to 'publisher-email@example.com'. |
 |  |  |
 
+### App manifest
+For `manifest.json` file, we will:
+1. Add a plugin node to the `manifest.json` file if a `manifest.json` file already exists in the output directory.
+
+```json
+"copilotExtensions": {
+  "plugins": [
+    {
+      "id": {plugin-name}
+      "file": "<generated_plugin_file>.json"
+    }
+  ]
+},
```

Comment:
we need to call out this manifest is NOT any of the previous manifest files we were talking about for the type parameter. Or this spec is going to get even more confusing.

Reply:
Done!